packages/opentelemetry-exporter-collector/src/types.ts

```diff
@@ -310,9 +310,11 @@ export namespace opentelemetryProto {
 export class CollectorExporterError extends Error {
   readonly code?: number;
   readonly name: string = 'CollectorExporterError';
+  readonly data?: string;
```

Comment:
Nothing is done with data field? Should we use a custom `toString` here?

Reply:
I actually though we needed to add a custom `toString` method too but its not needed, if you `console.log` this error it will also print both `code` and `data`